addon/ng2/commands/init.js

```diff
@@ -57,6 +59,14 @@ module.exports = Command.extend({
       });
     }
 
+    if (commandOptions.linkCli) {
```

Comment:
You should create that closer to where you use it. It's also weird (and will be invalid) to use `var linkCli` inside a scope block.

Also, use `let` for local variables to catch these.